srcpkgs/python3-sphinx-automodapi/template

```diff
@@ -1,16 +1,20 @@
 # Template file for 'python3-sphinx-automodapi'
 pkgname=python3-sphinx-automodapi
-version=0.13
-revision=4
+version=0.16.0
+revision=1
 build_style=python3-module
+# Something wrong with relative filepaths
+make_check_args="--deselect sphinx_automodapi/tests/test_automodapi.py::test_am_replacer_cython
+ --deselect sphinx_automodapi/tests/test_automodsumm.py::test_ams_cython"
 hostmakedepends="python3-setuptools"
 depends="python3-Sphinx"
+checkdepends="python3-wheel python3-Sphinx python3-pytest graphviz"
```

Comment:
Why it needs `wheel` in `checkdepends`?

Reply:
My mistake, it is not needed. Added it while testing but forgot to remove